Makefile

```diff
@@ -13,3 +13,22 @@ uninstall:
 	rm -rf $(DESTDIR)$(PREFIX)/bin/ani-cli
 
 .PHONY: all install uninstall
+
+#termux installation 
+TERMUX_BIN := /data/data/com.termux/files/usr/bin
+
+install-termux-mpv:
+	# copies ani-cli file to /data/data/com.termux/usr/bin/ani-cli, which should be in path
+	cp ani-cli $(TERMUX_BIN)/ani-cli
+	# marks ani-cli executable
+	chmod 0755 $(TERMUX_BIN)/ani-cli
+	# creating mpv file
+	@echo 'am start --user 0 -a android.intent.action.VIEW -d "$$2" -e "http-header-fields" "$$1" -n is.xyz.mpv/.MPVActivity' > $(TERMUX_BIN)/mpv
+	# marks mpv executable
+	chmod +x $(TERMUX_BIN)/mpv
+	# creating .cache folder
+	mkdir -p $(HOME)/.cache
+
+uninstall-termux-mpv:
+	rm -rf $(TERMUX_BIN)/ani-cli
+	rm -rf $(TERMUX_BIN)/mpv
```

Comment:
Why should this be in master?
Just implement the modified make file in your fork, as referenced in your Readme changes

Reply:
it has a error if i use the `make` 
![Screenshot_2022-01-18-21-04-39-62](https://user-images.githubusercontent.com/38305533/149942851-39a39f4e-ee0e-4a6d-91f0-99792d074eb7.png)
and if I change the PREFIX to fix the error the other os may have the same error so I just added a new line for termux installation

Reply:
oh sorry i not good in english

you can just add the termux installation in README.md
you don't need to add this changes
only add this changes if you'd like to change the git clone in README.md

Reply:
> Why should this be in master? Just implement the modified make file in your fork, as referenced in your Readme changes

I think he doesn't know about creating and managing branch.. so @ura43 can you create seperate android branch and push it??

Reply:
Sure @CoolnsX I hope I get to it sometime today